backend/case/src/main/java/org/pucar/dristi/util/CaseUtil.java

```diff
@@ -1,20 +1,34 @@
 package org.pucar.dristi.util;
 
 import java.security.SecureRandom;
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.util.Map;
+import java.util.UUID;
 
+import org.pucar.dristi.repository.CaseRepository;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 
+import static org.pucar.dristi.config.ServiceConstants.COURTID;
+import static org.pucar.dristi.config.ServiceConstants.ZONE_ID;
+
 @Component
 public class CaseUtil {
     private static final String CHARACTERS = "0123456789";
     private static final SecureRandom random = new SecureRandom();
 
+    @Autowired
+    private CaseRepository caseRepository;
+
     public String getCNRNumber(String fillingNumber, String state, String district, String establishmentCode) {
         String cnrNumber;
         String[] resp = fillingNumber.split("-");
-        String sequenceNumber = resp[resp.length-1];
-        String year = resp[resp.length-2];
-        cnrNumber = state+district+establishmentCode+"-"+sequenceNumber+"-"+year;
+        String sequenceNumber = resp[resp.length - 1];
+        String year = resp[resp.length - 2];
+        cnrNumber = state + district + establishmentCode + "-" + sequenceNumber + "-" + year;
 
```

Comment:
Sequence number is supposed to come from Database and is not the same as that used for Filing number. Please refer to the sheet again and the comments column 

https://docs.google.com/spreadsheets/d/1HPBO1MB62iDAP3pcvl2LWUuVcYyNBR20etZtWcxPIFU/edit?gid=0#gid=0

Reply:
This is not being used